Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean

```diff
@@ -536,6 +536,18 @@ theorem measure_closure_of_null_frontier {μ : Measure α'} {s : Set α'} (h : 
   measure_congr (closure_ae_eq_of_null_frontier h)
 #align measure_closure_of_null_frontier measure_closure_of_null_frontier
 
+instance separatesPoints_of_opensMeasurableSpace_of_t0Space [T0Space α] :
```

Comment:
Our naming convention wants that name instead:
```suggestion
instance separatesPointsOfOpensMeasurableSpaceOfT0Space [T0Space α] :
```
I think it's quite unreadable and yours would be better, but that discussion should happen elsewhere and for now we should follow the naming convention.